### DISCUSSION TITLE
Change config file type

### DISCUSSION THREAD

**Discussion by DinhHuy2010** [ThioJoe/YT-Spammer-Purge#422] [Ideas]
Can you change the config file type from ini to json? I really like json.

**Comment by ThioJoe** [comment]
It would require rewriting many hundreds of lines for no benefit, so no

**Reply by DinhHuy2010** [reply]
but I like to import settings like windows to mac so using json to import settings from any os.

**Reply by ThioJoe** [reply]
I don't see how changing the config to json would even make a difference with that

**Reply by DinhHuy2010** [reply]
wait what?

**Reply by ThioJoe** [reply]
> wait what?

I'm saying nothing you've posted makes any sense
